ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-detail.html

```diff
@@ -54,7 +54,7 @@
                                 <span class="column-title column-title-bottom">Alerts</span>
                             </div>
                         <div flex="20" layout="column" hide show-gt-md class="item-column md-list-item-text">
-                            <span class="item-title">{{component.checkDate | date : "MM/dd/yyyy 'at' h:mm"}}</span>
+                            <span class="item-title">{{'service-component-detail.time-locale' | translate:{date:component.checkDate} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:component.checkDate} }}</span>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.